### ISSUE TITLE
5.2 travis tests failing

### ISSUE THREAD

**Issue by suyograo** [elastic/logstash#6551]
Seems to be consistently failing at these tests:

```
spec ./specs/cli/install_spec.rb:40 # CLI > logstash-plugin install pack without internet connection (linux seccomp wrapper) successfully install the pack
rspec ./specs/cli/remove_spec.rb:28 # CLI > logstash-plugin remove without internet connection (linux seccomp wrapper) when no other plugins depends on this plugin successfully remove the plugin
rspec ./specs/cli/remove_spec.rb:43 # CLI > logstash-plugin remove without internet connection (linux seccomp wrapper) when other plugins depends on this plugin refuses to remove the plugin and display the plugin that depends on it.
rspec ./specs/cli/prepare_offline_pack_spec.rb:73 # CLI > logstash-plugin prepare-offline-pack create a pack with a locally installed .gem successfully create a pack
rspec ./specs/cli/prepare_offline_pack_spec.rb:23 # CLI > logstash-plugin prepare-offline-pack creating a pack for specific plugins successfully create a pack
rspec ./specs/cli/prepare_offline_pack_spec.rb:42 # CLI > logstash-plugin prepare-offline-pack create a pack from a wildcard successfully create a pack
```

https://travis-ci.org/elastic/logstash/jobs/191847302

**Comment by ph** [comment]
Looking at this, seems all the same root cause.

**Comment by ph** [comment]
My bad I've only at the test in the description, but if we look earlier the `rake test:core` throw this error.



```
Gem::LoadError: You have already activated jar-dependencies 0.3.9, but your Gemfile requires jar-dependencies 0.3.8. Prepending `bundle exec` to your command may solve this.
/home/travis/build/elastic/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/runtime.rb:34:in `setup'
/home/travis/build/elastic/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/runtime.rb:19:in `setup'
/home/travis/build/elastic/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler.rb:122:in `setup'
/home/travis/build/elastic/logstash/lib/bootstrap/bundler.rb:64:in `setup!'
/home/travis/build/elastic/logstash/rakelib/test.rake:21:in `(root)'
/home/travis/build/elastic/logstash/vendor/bundle/jruby/1.9/gems/rake-12.0.0/exe/rake:27:in `(root)'
Tasks: TOP => test:core => test:setup
(See full trace by running task with --trace)

**Comment by ph** [comment]
I've cleared jarvis cache and retrigged the build.

**Comment by ph** [comment]
I have a theory, I haven't been able to reproduce it locally yet.

Logstash 5.2's lock is set to jar-dependencies 0.3.8,  (we need to update it to fix http_proxy's issues) qa/integration also uses `jar-dependencies`, but don't have a specific version it will pickup the latest which is 0.3.9

Somehow, the 0.3.9 get activated before the 0.3.8 and it makes the whole thing fails.
We don't have this issue in the master branch since we always create a new lock file with the latest version.

The easiest fix it to bump to 0.3.9. I will wrap a PR for that to see if it makes Travis happy.


**Other ideas are:**

Use a path for the qa/integration test dependencies?
See if wrapping the call with bundle exec solves the issues.

**Comment by ph** [comment]
Fixed by #6556
